plugin/pkg/admission/broker/authsarcheck/admission_test.go

```diff
@@ -43,7 +43,7 @@ func newHandlerForTest(kubeClient kubeclientset.Interface) (admission.Interface,
 	if err != nil {
 		return nil, kf, err
 	}
-	pluginInitializer := scadmission.NewPluginInitializer(nil, nil, kubeClient, kf)
+	pluginInitializer := scadmission.NewPluginInitializer(nil, nil, kubeClient, kf, nil, nil)
```

Comment:
We don't seem to have any integration tests to make sure that these new arguments and changes in pluginInitializer don't break anything.
@jpeeler can you review please?